components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -40,6 +40,10 @@ let fetchNewClassIdRulesTimeoutId: number | undefined
 const queriedIds = new Set<string>()
 const queriedClasses = new Set<string>()
 
+const notYetQueriedElements: Array<(Element[] | NodeListOf<Element>)> = []
+
+const classIdWithoutHtmlOrBody = '[id]:not(html):not(body),[class]:not(html):not(body)'
```

Comment:
do we need `not(html):not(body)` for ids?

Reply:
Yes, see https://github.com/gorhill/uBlock/blob/master/src/js/contentscript.js#L1186-L1200

also related: https://github.com/brave/brave-browser/issues/17695